conf.yaml

```
@@ -131,6 +131,9 @@ contents:
               -
                 repo:   elasticsearch
                 path:   docs/reference
+              -
+                repo:   kibana
+                path:   docs
```

This is OK as is, but if we don't want to rebuild the glossary anytime *any* of the Kibana docs change, we can use this instead:
```suggestion
                path:   docs/glossary.asciidoc
```

Do I understand that correctly, @nik9000 ?

---

Ah, if that's the case I'll make the same change to the Glossary's elasticsearch dependency. It only cares about one file there too.